docs/source/en/guides/collections.md

```diff
@@ -58,6 +58,45 @@ A `note` can also be attached to the item. This is useful to add additional info
 
 In addition to these base attributes, returned items can have additional attributes depending on their type: `author`, `private`, `lastModified`, `gated`, `title`, `likes`, `upvotes`, etc. None of these attributes are guaranteed to be returned.
 
+## List collections
+
+We can also get multiple collections with [`list_collection`]. It allows us to retrieve collections and filter them with certain parameters. Let's list all collections of the user `teknium`.
+```py
+>>> from huggingface_hub import list_collections
+
+>>> collections = list_collections(owner="teknium")
+```
+This returns an iterable of `Collection` objects. We can iterate over it to print number of upvotes for each collections.
```

Comment:
```suggestion

This returns an iterable of `Collection` objects. We can iterate over them to print, for example, the number of upvotes for each collection.

```

(adding some newlines to help the doc builder parse correctly the markdown file)